main.go

```diff
@@ -0,0 +1,42 @@
+package main
+
+import (
+	"os"
+
+	"github.com/bitrise-steplib/bitrise-step-android-instrumented-test/step"
+
+	"github.com/bitrise-io/go-steputils/v2/stepconf"
+	"github.com/bitrise-io/go-utils/v2/command"
+	"github.com/bitrise-io/go-utils/v2/env"
+	"github.com/bitrise-io/go-utils/v2/log"
+)
+
+func main() {
+	os.Exit(run())
+}
+
+func run() int {
+	logger := log.NewLogger()
+	instrumentedTestRunner := createTestRunner(logger)
+	exitCode := 0
+
+	config, err := instrumentedTestRunner.ProcessConfig()
+	if err != nil {
+		logger.Errorf(err.Error())
+		exitCode = 1
```

Comment:
we should terminate the execution here, if the step failed to prepare run configuration based on the inputs, it doesn't make sense to call Step.Run.